daisy_workflows/image_build/enterprise_linux/kickstart/el7-post.cfg

```diff
@@ -1,10 +1,22 @@
 # Google Compute Engine kickstart config for Enterprise Linux 7.
-%post --log=/dev/ttyS0
-set -x
+%onerror
+echo "BuildFailed!" > /dev/ttyS0
```

Comment:
i still don't think we should be using daisy-isms in here. leaving this code independent is cleaner if possible.

Reply:
also, el6-post doesn't have this %onerror section, is it not supported on el6?

Reply:
No unfortunately el6 doesn't support this wonderful feature :( So... basically it'll just crash but the logs will still be there and you can see why.

Reply:
Not a daisy-ism just a message for the fail conditions.

Reply:
BuildFailed is a dasiy-ism, though. but i guess it's not too bad, so long as the behavior of the ks rules doesn't depend on being run by daisy.